keyboards/pos78/readme.md

```diff
@@ -0,0 +1,18 @@
+# pos78
```

Comment:
Can you reformat this file to match our [template](https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md)?

Reply:
One of the other reviewers suggested this format...

Reply:
Yeah that was my bad, sorry.